extracted/konan.serializer/src/org/jetbrains/kotlin/konan/utils/KonanFactories.kt

```diff
@@ -0,0 +1,51 @@
+package org.jetbrains.kotlin.konan.utils
+
+import org.jetbrains.kotlin.descriptors.konan.KonanModuleDescriptorFactory
+import org.jetbrains.kotlin.descriptors.konan.impl.KonanModuleDescriptorFactoryImpl
+import org.jetbrains.kotlin.serialization.konan.KonanDeserializedModuleDescriptorFactory
+import org.jetbrains.kotlin.serialization.konan.KonanDeserializedPackageFragmentsFactory
+import org.jetbrains.kotlin.serialization.konan.KonanResolvedModuleDescriptorsFactory
+import org.jetbrains.kotlin.serialization.konan.impl.KonanDeserializedModuleDescriptorFactoryImpl
+import org.jetbrains.kotlin.serialization.konan.impl.KonanDeserializedPackageFragmentsFactoryImpl
+import org.jetbrains.kotlin.serialization.konan.impl.KonanResolvedModuleDescriptorsFactoryImpl
+
+/**
+ * The default Kotlin/Native factories.
+ */
+@Suppress("MemberVisibilityCanBePrivate")
```

Comment:
Is this suppress really needed?

Reply:
I've added it just because otherwise IDEA showed me annoying warn that both functions declared in this object are public but could be private. Will remove it, without it the code will look cleaner.